oracle/price-feeder/oracle/oracle.go

```diff
@@ -391,8 +393,8 @@ func (o *Oracle) GetParamCache(ctx context.Context, currentBlockHeight int64) (o
 func (o *Oracle) GetParams(ctx context.Context) (oracletypes.Params, error) {
 	grpcConn, err := grpc.Dial(
 		o.oracleClient.GRPCEndpoint,
-		// the Cosmos SDK doesn't support any transport security mechanism
-		grpc.WithInsecure(),
+		grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: true})),
+		grpc.WithBlock(),
```

Comment:
Could you give context around the discussion in this PR so future readers can reference it?

Reply:
does it make sense to block?

Reply:
Block shouldn't be necessary